tfhe/src/integer/server_key/radix_parallel/tests_cases_comparisons.rs

```diff
@@ -0,0 +1,1946 @@
+                // We don't use PARAM_MESSAGE_3_CARRY_3_KS_PBS,
+                // as smart test might overflow values
+                // and when using 3_3 to represent 256 we actually have more than 256 bits
+                // of message so the overflow behaviour is not the same, leading to false negatives
+                PARAM_MESSAGE_4_CARRY_4_KS_PBS
+            });
+
+            create_parametrized_test!([<integer_default_ $comparison_name _parallelized_256_bits>]
+            {
+                PARAM_MESSAGE_2_CARRY_2_KS_PBS,
+                // We don't use PARAM_MESSAGE_3_CARRY_3_KS_PBS,
+                // as default test might overflow values
+                // and when using 3_3 to represent 256 we actually have more than 256 bits
+                // of message so the overflow behaviour is not the same, leading to false negatives
+                PARAM_MESSAGE_4_CARRY_4_KS_PBS
+            });
+        }
+    };
+}
+
+use crate::shortint::parameters::{
+    PARAM_MESSAGE_2_CARRY_2_KS_PBS, PARAM_MESSAGE_3_CARRY_3_KS_PBS, PARAM_MESSAGE_4_CARRY_4_KS_PBS,
+};
+
+define_comparison_test_functions!(eq);
+define_comparison_test_functions!(ne);
+define_comparison_test_functions!(lt);
+define_comparison_test_functions!(le);
+define_comparison_test_functions!(gt);
+define_comparison_test_functions!(ge);
+
+//================
+// Min
+//================
+
+#[test]
+fn test_unchecked_min_256_bits_param_message_2_carry_2_ks_pbs() {
+    test_unchecked_min_256_bits(
+        crate::shortint::parameters::PARAM_MESSAGE_2_CARRY_2_KS_PBS,
+        4,
+    )
+}
+
+#[test]
+fn test_unchecked_min_256_bits_param_message_3_carry_3_ks_pbs() {
+    test_unchecked_min_256_bits(
+        crate::shortint::parameters::PARAM_MESSAGE_3_CARRY_3_KS_PBS,
+        2,
+    )
+}
+
+#[test]
+fn test_unchecked_min_256_bits_param_message_4_carry_4_ks_pbs() {
+    test_unchecked_min_256_bits(
+        crate::shortint::parameters::PARAM_MESSAGE_4_CARRY_4_KS_PBS,
+        2,
+    )
+}
+
+#[test]
+fn test_unchecked_min_parallelized_256_bits_param_message_2_carry_2_ks_pbs() {
+    test_unchecked_min_parallelized_256_bits(
+        crate::shortint::parameters::PARAM_MESSAGE_2_CARRY_2_KS_PBS,
+        4,
+    )
+}
+
+#[test]
+fn test_unchecked_min_parallelized_256_bits_param_message_3_carry_3_ks_pbs() {
+    test_unchecked_min_parallelized_256_bits(
+        crate::shortint::parameters::PARAM_MESSAGE_3_CARRY_3_KS_PBS,
+        2,
+    )
+}
+
+#[test]
+fn test_unchecked_min_parallelized_256_bits_param_message_4_carry_4_ks_pbs() {
+    test_unchecked_min_parallelized_256_bits(
+        crate::shortint::parameters::PARAM_MESSAGE_4_CARRY_4_KS_PBS,
+        2,
+    )
+}
+
+#[test]
+fn test_min_parallelized_256_bits_param_message_2_carry_2_ks_pbs() {
+    test_min_parallelized_256_bits(
+        crate::shortint::parameters::PARAM_MESSAGE_2_CARRY_2_KS_PBS,
+        4,
+    )
+}
+
+// #[test]
+// fn test_min_parallelized_256_bits_param_message_3_carry_3_ks_pbs() {
```

Comment:
commented out ? add a reason

Reply:
They were already commented, its the same reason as for non unchecked comparisons in 3_3

Reply:
comment added